noir-projects/noir-contracts/contracts/app_subscription_contract/src/main.nr

```diff
@@ -34,7 +34,8 @@ contract AppSubscription {
 
     #[aztec(private)]
     fn entrypoint(payload: DAppPayload, user_address: AztecAddress) {
-        assert(context.msg_sender().to_field() == 0);
+        // Default msg_sender for entrypoints is now Fr.max_value rather than 0 addr (see #7190 & #7404)
+        assert(context.msg_sender().to_field() == 0 - 1);
```

Comment:
Will be nice to create a constant so that it's easier to find where MAX_FIELD is used!

Reply:
Done! It's called `MAX_FIELD_VALUE` across the repo